### PR TITLE
[Task] Use mkdocs git submodule in oaas docs SA 5305

### PR DESCRIPTION
**Pull request by RSAK56**
# Description
Updated oaas docs with theme common
Fixes # (issue)
UI/UX
# Type of change
Enhancement
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] UI/UX modification
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: theme_override/resources/assets/images/logo-dark.png
Comment (ufateh) [inline theme_override/resources/assets/images/logo-dark.png:1]: dark and light logo look the same, why repeat?

File: theme_override/mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - pymdownx.details
+  - pymdownx.superfences:
+      custom_fences:
+        - name: mermaid
+          class: mermaid
+          format: !!python/name:pymdownx.superfences.fence_code_format
+  - attr_list
+  - pymdownx.emoji:
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
```
Comment (ufateh) [inline theme_override/mkdocs.yml:27]: some props look like already exist in common. double check.

File: prepare_theme.sh
Comment (ufateh) [inline prepare_theme.sh:1]: this script already in common, use it from there. update common readme on how to use it in child projects.

Comment (ufateh) [reply prepare_theme.sh:1]: no need to copy here.

File: mkdocs.yml
Comment (ufateh) [inline mkdocs.yml:1]: this file should have been removed. use gitignore file from mto-docs.

File: custom_theme/404.html
Comment (ufateh) [inline custom_theme/404.html:1]: custom theme have its own 404 and main?
make sure you do not repeat whats in common.

File: theme_override/resources/assets/images/logo-dark.png
Comment (RSAK56) [reply theme_override/resources/assets/images/logo-dark.png:1]: The dark logo will be modified in the future with a updated one so will just need to replace the file with the updated image

File: theme_override/mkdocs.yml (hunk 1 above)
Comment (RSAK56) [reply theme_override/mkdocs.yml:27]: Same: The issue with the merge script still exists for list properties, there's another ticket open for this issue that was raised earlier. We need to look at that, for theme common in general.

File: custom_theme/404.html
Comment (RSAK56) [reply custom_theme/404.html:1]: These files are already removed kindly recheck the commit

File: mkdocs.yml
Comment (RSAK56) [reply mkdocs.yml:1]: Updated gitignore not to trace this file

File: prepare_theme.sh
Comment (RSAK56) [reply prepare_theme.sh:1]: Tried the fix has some issue with the directory we can check on it in the next releases

File: theme_override/resources/assets/images/logo-dark.png
Comment (ufateh) [inline theme_override/resources/assets/images/logo-dark.png:1]: this logo is same as logo-light, why same file twice?

File: theme_override/mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+site_name: OpenShift as a Service
+site_url: https://docs.stakater.com/oaas/
+repo_url: https://github.com/stakater/oaas-docs
+
+theme:
```
[the remark this answers is not in the document]
Comment (RSAK56) [reply theme_override/mkdocs.yml]: Added back the content path here

File: theme_override/resources/assets/images/logo-dark.png
Comment (RSAK56) [reply theme_override/resources/assets/images/logo-dark.png:1]: The dark logo will be modified in the future with a updated one so will just need to replace the file with the updated image

File: theme_override/mkdocs.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ufateh) [reply theme_override/mkdocs.yml]: where?

**Review by ufateh — APPROVED** [review]

File: theme_override/mkdocs.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (RSAK56) [reply theme_override/mkdocs.yml]: In the file changes, this one's outdated

**Pull request merged**